<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationPanelWidget</class>
 <widget class="QWidget" name="StationPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="stationTypes">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="recent">
        <property name="text">
         <string>Recent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="friends">
        <property name="text">
         <string>Friends</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="neighbours">
        <property name="text">
         <string>Neighbours</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="artists">
        <property name="text">
         <string>Artists</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tags">
        <property name="text">
         <string>Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="recentPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="recentList" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="friendsPage">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="friendsList" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="friendsFilter"/>
         </item>
         <item>
          <widget class="QPushButton" name="friendsSort">
           <property name="text">
            <string>sort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="neighboursPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="neighboursList" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="artistsPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="artistsList" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tagsPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="tagsList" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header>../widgets/StationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
